gallery_dl/extractor/batoto.py

```diff
@@ -54,7 +54,7 @@ def metadata(self, page):
 
         match = re.match(
             r"(?:Volume\s+(\d+) )?"
-            r"\w+\s+(\d+)(.*)", info)
+            r"[Cc]hapter\s*(\d+)(\.\d+)?", info)
```

Comment:
```suggestion
            r"[Cc]hapter\s*(\d+)([\w.]*)", info)
```

There might be minor chapter versions in a non-standard format, like `Chapter 123a`, `Chapter 123b`, ... I've also had `[^\s:]*` or even `\S*` in mind.

And then there might be non-English chapter identifiers, which is probably why I've used `\w+` instead of an explicit `[Cc]hapter`. Looking at a few examples, the explicit form is definitely better but we should also add the full, unparsed chapter string value as an alternative when regular parsing fails.

``` patch
diff --git a/gallery_dl/extractor/batoto.py b/gallery_dl/extractor/batoto.py
index 19261f8c..e3aa55ec 100644
--- a/gallery_dl/extractor/batoto.py
+++ b/gallery_dl/extractor/batoto.py
@@ -72,6 +72,7 @@ class BatotoChapterExtractor(BatotoBase, ChapterExtractor):
             "volume"       : text.parse_int(volume),
             "chapter"      : text.parse_int(chapter),
             "chapter_minor": minor,
+            "chapter_string": info,
             "chapter_id"   : text.parse_int(self.chapter_id),
             "date"         : text.parse_timestamp(extr(' time="', '"')[:-3]),
         }
```